### PR TITLE
Fixing build warnings

### PR DESCRIPTION
**Pull request by BrennanConroy**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -204,7 +204,7 @@
     <!-- Packages from 2.1/2.2 branches used for site extension build -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing.20163.14</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing-20113-19</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
```
Comment (Tratcher) [inline eng/Versions.props:207]: Going back in time?

Comment (BrennanConroy) [reply eng/Versions.props:207]: I'm not sure where these versions are coming from, was going to see any of the build folks knew.

Comment (dougbu) [reply eng/Versions.props:207]: Could you give me an example of a warning related to using the 3.1.3-servicing.20163.14 version?

Comment (BrennanConroy) [reply eng/Versions.props:207]:
```
##[warning]src\SiteExtensions\LoggingAggregate\src\Microsoft.AspNetCore.AzureAppServices.SiteExtension\Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj(0,0): warning NU1603: Microsoft.AspNetCore.AzureAppServices.SiteExtension depends on Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 (>= 3.1.3-servicing.20163.14) but Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 3.1.3-servicing.20163.14 was not found. An approximate best match of Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 3.1.3-servicing-20113-19 was resolved.
```
https://dev.azure.com/dnceng/internal/_build/results?buildId=610167&view=logs&j=1b89928a-2219-5ef9-602f-f95beb3da4dc&t=39263dad-2be6-5f40-bde4-1bab0d7931b4&l=862

Comment (dougbu) [reply eng/Versions.props:207]: Looks more like a feed we need to build correctly has been removed from NuGet.config. @wtgodbe❔

Comment (dougbu) [reply eng/Versions.props:207]: Just double-checked. Undo changes to this file and instead add missing feed to NuGet.config:
``` xml
 <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
```
Feel free to include a comment so we don't forget 😸

Comment (BrennanConroy) [reply eng/Versions.props:207]: Thanks 👍

File: NuGet.config [hunk 2]
```diff
@@ -11,5 +11,7 @@
     <add key="aspnetcore-dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
     <add key="aspnetcore-tools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
     <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
+    <!-- Used for the SiteExtension 3.1 bits that are included in the 5.0 build -->
+    <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
```
Comment (dougbu) [inline NuGet.config:15]: 👍

**Review by dougbu — APPROVED** [review]

**Pull request merged**